### ISSUE TITLE
Can't write information on the invoicing screen via the clickthrough on the project

### ISSUE THREAD

**Issue by Thomasjanssens001** [be-pongit/confac#222]
## **Context:**
Op het facturatiescherm zijn er 3 manieren om commentaar te schrijven. Slechts 2 van de 3 hebben een knop "bewaren".

## **Environment:**
Test

## **Steps To Reproduce:**
1. klik op de projectknop op eht facturatiescherm
2. klik op de commentarenknop (spraakbubbel)
3. schrijf tekst in de pop-up
4. ga uit de tekstbubbel (via "X" of "Sluiten")
5. refresh
6. navigeer opnieuw naar de spraakbubbel -> tekst is **niet** bewaard gebleven 

## **Expected Result:**
1. klik op de projectknop op eht facturatiescherm
2. klik op de commentarenknop (spraakbubbel)
3. schrijf tekst in de pop-up
4. ga uit de tekstbubbel (via "X" of "Sluiten")
5. refresh
6. navigeer opnieuw naar de spraakbubbel -> tekst is bewaard gebleven 

## **Severity:** 
High

## **Prio:**
High

**Comment by Laoujin** [comment]
Ik denk dat in je foto's je nu 2 pijlen hebt naar de 2 commentaar velden die werken maar geen enkele naar het commentaar veld dat NIET werkt :)

**Comment by Thomasjanssens001** [comment]
Nee, ik zie wat ik verkeerd gedaan heb. Ik ben een foto vergeten bijvoegen.

**Comment by Runekid** [comment]
Hi @Thomasjanssens001 @Laoujin 

Ik heb wat rondgekeken in de code en de bewaarknop is hardcoded op disabled gezet.
commit met de change https://github.com/itenium-be/confac/commit/082709177a28356ba80cc1408f027ad521a0d360

De bedoeling van die notes knop is geloof ik enkel om de notes van het project te tonen en niet aan te passen via daar. Die notes kan je ook terugvinden op het project detail view waar je het wel kan aanpassen.
![image](https://user-images.githubusercontent.com/46999213/229525072-80d4132f-c57e-459d-bd4c-c8b6457b4459.png)
![image](https://user-images.githubusercontent.com/46999213/229525238-a5a0b1f8-68b1-4f3c-ac1f-b2f84ae0d5e2.png)

**Comment by Laoujin** [comment]
Ah juist, dat was een heel probleem ofzo om dat van daar te bewaren en daarmee dat ik het readonly gemaakt heb.

As designed dus 😀

**Comment by Thomasjanssens001** [comment]
Kunnen we anders niet de naam van dat veld lichtjes veranderen om het wat duidelijker te maken?
-> Eventuele opties waar ik aan dacht: Projectnotities (read-only); Projectnotities (non-editable)

Een andere optie is eventueel de layout van dat tekstveld te veranderen zodat het er niet meer uitziet alsof je het kan editeren of het veld non-editable te maken.

We zouden ook beide van bovenstaande opties kunnen implementeren als dit mogelijk is.

Thoughts?

**Comment by Brauwer** [comment]
Fixed
